.editorconfig

```diff
@@ -11,3 +11,4 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 indent_style = space
 indent_size = 2
+preferred_line_length = 80
```

Comment:
i've been using 120 -- 80 is a bit of an old standard, and it's rare to find a screen that small now.
:elephant:

Reply:
We should match our rubocop settings too -- so actually 120 is a better idea i think